### ISSUE TITLE
Push to github button is missing after latest update

### ISSUE THREAD

**Issue by hbadri3** [getinsomnia/insomnia#5713]
### Expected Behavior

After syncing with github repo.
I should be able to commit and push.

### Actual Behavior

The commit is telling me nothing to commit.
The push button is not there

<img width="446" alt="Screenshot 2023-01-23 at 14 18 43" src="https://user-images.githubusercontent.com/94041573/214027438-67937c50-3e95-491f-9594-f7fb209ffffd.png">

This is the version I have updated for lately on mac:

<img width="321" alt="Screenshot 2023-01-23 at 14 20 16" src="https://user-images.githubusercontent.com/94041573/214027646-dfd9dd28-2497-428f-bcbc-16b0cb432948.png">

### Reproduction Steps

- Sync you github repo with authorization and eveything
- Try to make some changes and commit

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

 2022.7.5

### What operating system are you using?

macOS

### Operating System Version

Ventur 13.1 (22C65)

### Installation method

download from insomnia.rest

**Comment by filfreire** [comment]
@hbadri3 do you see the push button if you remove and re-clone the repository?

It's also weird that in your case it's defaulting to `master` branch but with a history of 0 commits. I would suggest trying to do a clean git clone first.

Another thing that might help is if you share any error console logs from the dev tools. To view these View > Toggle Developer tools, and then in the dev tools open the Console tab. When setting up and cloning the repository or when opening lookout for error messages.

**Comment by hbadri3** [comment]
@filfreire 
Appreciate your reply.
Actually, I have made several trials with a clean repo. I guess because of the update that was made to Insomnia lately. I have some collections already that I pulled from a previous repo so that I start fresh.
The authentication is successful.

<img width="611" alt="Screenshot 2023-01-29 at 09 21 48" src="https://user-images.githubusercontent.com/94041573/215308971-8a7bf9a7-4807-4e98-9626-c1ebfffe1f31.png">
